sdk/cloudmachine/Azure.CloudMachine/samples/AzureSImpleRAG/Pages/Error.cshtml.cs

```diff
@@ -0,0 +1,27 @@
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace AzureSimpleRAG.Pages;
+
+[ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+[IgnoreAntiforgeryToken]
+public class ErrorModel : PageModel
```

Comment:
does it need to be public?

Reply:
I believe this is common for ASPNET pages to be public